app/src/main/java/cm/aptoide/pt/ApplicationModule.java

```diff
@@ -1085,6 +1095,16 @@ AnalyticsBodyInterceptorV7 provideAnalyticsBodyInterceptorV7(
         .build();
   }
 
+  @Singleton @Provides @Named("retrofit-wallet") Retrofit providesV8SecondaryRetrofit(
```

Comment:
this does not depend on the wallet, it is a general v8 secondary retrofit instance. I would suggest "retrofit-v8-secondary".

Reply:
fixed